site/components/pages/Retirement/SingleRetirement/RetirementHeader/styles.ts

```diff
@@ -0,0 +1,20 @@
+import { css } from "@emotion/css";
+
+export const retirementHeader = css`
+  grid-column: main;
+  padding: 8rem 2.4rem;
+  background: url("/green_hills.png");
```

Comment:
it's annoying but we should use nextjs images as they are much better optimized...

Reply:
Done